team_project_access_integration_test.go

```diff
@@ -166,13 +166,15 @@ func TestTeamProjectAccessesAdd(t *testing.T) {
 	})
 
 	t.Run("with valid options for all custom TeamProject permissions", func(t *testing.T) {
+		skipUnlessBeta(t)
```

Comment:
Since this tests more than just the beta variable sets, I think it would be better to have a separate, skipUnlessBeta test specifically for variable sets. Otherwise, we won't be testing the other custom team project permissions.